arduino-core/src/processing/app/helpers/FileUtils.java

```diff
@@ -85,7 +85,7 @@ public static void recursiveDelete(File file) {
         recursiveDelete(current);
       }
     }
-    deleteIfExists(file);
+    file.delete();
```

Comment:
Since you deleted the `.exists()` check in `deleteIfExists`, could you still call that function instead of `file.delete()`? It uses a defensive approach checking if File is `null`